### PR TITLE
MSI-2084 Introduce tests for 'GetNearbySourcesByPostcode'

### PR DESCRIPTION
**Pull request by vadimjustus**
Introduces integration tests for `\Magento\InventoryInStorePickup\Model\DistanceProvider\Offline\GetNearbySourcesByPostcode` class.

### Fixed Issues (if relevant)
1. magento-engcom/msi#2084: Cover nearest inventory source selection with integration tests

### Manual testing scenarios (*)
1. Running integration tests for `InventoryInStorePickup` module

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by IvanPletnyov** [comment]
I've run this test, test was passed.

**Comment by maghamed** [comment]
hey @vadimjustus could you pleasee look at integration tests, looks like they are broken after your changes

File: app/code/Magento/InventoryInStorePickup/Test/_files/inventory_geoname.php [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+	('IT', '48032', 'Casola Valsenio', 'Emilia-Romagna', 'RA', 44.2244, 11.6249),
+	('IT', '19133', 'Laspezia', 'Liguria', 'SP', 44.0967, 9.7753),
+	('IT', '56038', 'Giardino', 'Toscana', 'PI', 43.5022, 10.6669),
+	('IT', '47834', 'Serbadone', 'Emilia-Romagna', 'RN', 43.8983, 12.636),
+	('IT', '63827', 'Pedaso', 'Marche', 'FM', 43.0984, 13.8408),
+	('IT', '41032', 'Motta', 'Emilia-Romagna', 'MO', 44.8254, 10.9974),
+	('IT', '45030', 'Ceneselli', 'Veneto', 'RO', 45.0138, 11.3694),
+	('IT', '09040', 'San Nicolo\' Gerrei', 'Sardegna', 'CA', 39.2071, 9.1074),
+	('IT', '80022', 'Arzano', 'Campania', 'NA', 40.9096, 14.2652),
+	('IT', '89020', 'Melicucca\'', 'Calabria', 'RC', 38.3031, 15.8815),
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryInStorePickup/Test/_files/inventory_geoname.php]: `'Melicucca\''` might be a typo

**Comment by ishakhsuvarov** [comment]
`continuous-integration/travis-ci/push` was launched by mistake and cancelled.

**Review by ishakhsuvarov — APPROVED** [review]

**Pull request merged**